cockroachcloud/cluster-overview-page.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Cluster Overview Page
+summary: How to use the Cluster Overview page to view cluster details on {{ site.data.products.serverless }}.
+toc: true
+---
+
+The **Cluster Overview** page displays key metrics about your {{ site.data.products.serverless }} cluster and individual nodes.
+
+## Cluster settings
+
+The **Cluster settings** panel displays the settings you chose during [cluster creation](create-a-serverless-cluster.html).
+
+| Field     | Description                                                                                             |
+|-----------|---------------------------------------------------------------------------------------------------------|
+| Cloud     | The cluster's [cloud provider](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region). |
+| Plan type | The plan type used to create the cluster.                                                               |
+| Region    | The cluster's [region](create-a-serverless-cluster.html#step-2-select-a-cloud-provider-region).         |
+
+## Usage this month
+
+The **Usage this month** panel displays your cluster usage statistics for the current month.
+
+| Field         | Description                                                                                                                                                                                                                                                                               |
+|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spend limit   | The spend limit you entered when [creating the cluster](create-a-serverless-cluster.html#step-3-enter-a-spend-limit). Click the **edit** icon to change the spend limit.                                                                                                                  |
+| Storage       | The amount of data currently stored in the cluster, and the total available storage capacity, before replication. Total available storage  is determined by your configured spend limit and available RUs. For details, see [CockroachCloud Architecture](architecture.html#performance). |
```

Comment:
One thing we might want to clarify is why it says "x / up to y" for Storage and Request units. I think it's not quite accurate to say "Total available storage is determined by your configured spend limit and available RUs" because it actually depends on the Burst RUs you've already used and how much you might use for the rest of the month. I guess I would just say "...determined by your spend limit and Burst RU usage," and vice versa the RU maximum is determined by your spend limit and storage usage.